<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>813</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mini C 分析程序v1.11</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>源码内容：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>11</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>当前文件：</string>
    </property>
   </widget>
   <widget class="QLabel" name="filename">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>721</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>541</width>
      <height>691</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="treetab">
     <attribute name="title">
      <string>语法树</string>
     </attribute>
     <widget class="QTextBrowser" name="treetext">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>511</width>
        <height>651</height>
       </rect>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="lineWrapMode">
       <enum>QTextEdit::NoWrap</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="codetab">
     <attribute name="title">
      <string>中间代码</string>
     </attribute>
     <widget class="QTextBrowser" name="codetext_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>511</width>
        <height>651</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="readbutton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string> 读取文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearbutton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string> 清空文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="parsebutton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>分析源码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="codetext">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>391</width>
      <height>651</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="treSavBtn">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>保存语法树</string>
    </property>
   </widget>
   <widget class="QPushButton" name="codesave">
    <property name="geometry">
     <rect>
      <x>664</x>
      <y>40</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string> 另存中间代码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="code_run">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>40</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>解释执行</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>990</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
